types/stylelint/index.d.ts

```diff
@@ -9,7 +9,9 @@ declare module 'stylelint' {
 	export type Severity = 'warning' | 'error';
 
 	type ConfigExtends = string | string[];
-	type Plugin = { default?: { ruleName: string; rule: Rule } } | { ruleName: string; rule: Rule };
+	export type Plugin =
```

Comment:
[suggest] Can you also add JSDOC, please?

```suggestion
	/**
	 * A Stylelint plugin.
	 */
	export type Plugin =
```